pkg/storageincentives/agent.go

```diff
@@ -27,10 +29,12 @@ const loggerName = "storageincentives"
 const (
 	DefaultBlocksPerRound = 152
 	DefaultBlocksPerPhase = DefaultBlocksPerRound / 4
+	MaxInt64              = 9223372036854775807
```

Comment:
The max int64 is already defined in the `math` package as `math.MaxInt64`. On top of that, the constant isn't used.